### ISSUE TITLE
After update to 2.5.9

### ISSUE THREAD

**Issue by lastsamurai26** [JulianPrieber/littlelink-custom#35]
SQLSTATE[HY000]: General error: 1 no such column: links.custom_css (SQL: select "links"."link", "links"."id", "links"."button_id", "links"."title", "links"."custom_css", "links"."custom_icon", "buttons"."name" from "links" inner join "buttons" on "buttons"."id" = "links"."button_id" where "user_id" = 3 order by "up_link" asc, "order" asc)

I hope I didn't do anything wrong but after the update I can't access my page anymore only the admin panel.

**Comment by JulianPrieber** [comment]
This is most likely due to a missing database entry. The last update added two new database entries, which you're probably missing. 

Assuming you are running with the default, SQLite database, try temporarily replacing your database with the one in the zip file found [here](https://github.com/JulianPrieber/littlelink-custom/files/8574933/database.zip), just to be sure. 

Please make a backup of your current database before doing that.

**Comment by lastsamurai26** [comment]
Yes, I saw it :/ new indexes have been created in the create links :-D dooof can't I update them somehow ? means I have to transfer my datasets

**Comment by JulianPrieber** [comment]
Honestly, I'm not even sure myself. Lemme try real quick.

**Comment by JulianPrieber** [comment]
In my testing, I was not able to add the entry with the usual migrate command `php artisan migrate`. After some digging online, it seems to be possible to add new columns to the database with migrations, but I'm not really sure how. 

I would probably just add the entries manually. Or rebuild the links from scratch.

**Comment by lastsamurai26** [comment]
I compared both databases with each other and copied all the data records from my own database as SQL and then copied them into the new database, including the users. 


Correct I have rebuilt the database manually with the "Update database".

**Comment by JulianPrieber** [comment]
About a month ago, I experimented with implementing this solution [here](https://github.com/codedge/laravel-selfupdater) and this would probably work for us, but currently there is a bug present in the code which makes using the thing impossible. This bug was pointed out multiple times by users, but the dev isn't actively responding to issues at the moment, so it might be awhile until this can finally be implemented.

**Comment by lastsamurai26** [comment]
ok, if this has already been reported but the developer does not fix it, does anyone know where the error lies so that this can be cloned and then fixed?

**Comment by JulianPrieber** [comment]
I tried a bit myself, but I can't really find the cause. This appears to be a simple formatting error that should be fairly easy to solve if you know what you're doing in PHP.

**Comment by lastsamurai26** [comment]
I'll have to take a look at it and see if I can find the problem.

**Comment by JulianPrieber** [comment]
This issue probably describes it best: https://github.com/codedge/laravel-selfupdater/issues/309. I'm fairly certain that this issue got introduced with a decently recent push, but I'm not entirely certain for the time frame.

The issue is that a function calling a certain URL with cURL has a syntax error, more precisely a simple "/" is missing between the version variable and the URL.
